packages/vuetify/src/components/VBanner/VBannerContent.ts

```diff
@@ -0,0 +1,24 @@
+// Components
+import VBannerThumbnail, { makeThumbnailProps } from './VBannerThumbnail'
+
+// Utilities
+import { defineComponent, h } from 'vue'
+import makeProps from '@/util/makeProps'
+
+export default defineComponent({
+  name: 'VBannerContent',
+
+  inheritAttrs: false,
+
+  props: makeProps(makeThumbnailProps()),
+
+  render () {
```

Comment:
If you really want these as separate components then I think you should at least use `setup` instead of `render`. I don't think we should be mixing old style and composition style among our code. As soon as you need state you will need to switch anyway. And moving code from one style to another will require rewrites